tests/runtest.cmd

```diff
@@ -477,6 +482,23 @@ goto :eof
     echo CoreCLR tests restored from %TESTS_REMOTE_URL% > %TESTS_SEMAPHORE%
     exit /b 0
 
+:TextExtRepoCoreCLRFramework
```

Comment:
Nit - can this be a typo; shouldn't this be rather spelled "TestExtRepoCoreCLRFramework" (Te _s_ t instead of Te _x_ t)?

Reply:
I agree, looks like a small typo